internal/keycloakb/accountingclient.go

```diff
@@ -27,6 +27,7 @@ type accountingClient struct {
 // HTTPClient interface
 type HTTPClient interface {
 	Get(data interface{}, plugins ...plugin.Plugin) error
+	Post(data interface{}, plugins ...plugin.Plugin) (string, error)
```

Comment:
May be I added this but did not use... Could you try to remove it?

Reply:
Good point ! Removed